shim/tests/mod.rs

```diff
@@ -1,5 +1,69 @@
 use assert_cmd::Command;
 
+static TURBO_HELP: &str = "turbo 
```

Comment:
Commented on #2386, but I generally prefer to keep UI snapshot tests in something like cram/prysk that has better support for updating snapshots support for diffing snapshots.